### DISCUSSION TITLE
Github raw files

### DISCUSSION THREAD

**Discussion by Mr-Stark-RDJ** [community/community#61569] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

After some time the raw file URL gives 404 ERROR and we have to get a new URL again and again is there any way to get a raw URL which works like forever or longer?

**Comment by pranjal-barnwal** [comment]
You can copy any image and paste it directly in **Markdown-based editor** in GitHub like even here and it will auto generate a link for that image & you can then use it anywhere.

Here's a sample for it with a random image:
![Random Github image](https://github.com/community/community/assets/71400605/d6577ff0-c763-405e-80b1-cf594d84ae97)

Try it yourself.
If it worked then please make sure to mark this as answer.
